### DISCUSSION TITLE
Windows 端应该如何更新？

### DISCUSSION THREAD

**Discussion by akatsukiro** [lollipopkit/flutter_server_box#191] [Q&A] [answered]
软件提示我应该更新，正常下载最新版替换所有文件后打开依然提示需要更新，且版本号没有改变。新建文件夹后放入最新版也依然提示需要更新。删除 Documents 目录下所有相关文件似乎也没起到效果。看压缩包文件，hash和时间新版和旧版确实不同。

所以我应该怎么做才能正确更新软件呢？

**Comment by lollipopkit** [comment, marked as answer]
貌似是我打包的问题，等会儿解决

**Comment by lollipopkit** [comment]
现在试试？

**Reply by akatsukiro** [reply]
可以了，十分感谢。
